packages/vx-tooltip/test/TooltipWithBounds.test.tsx

```diff
@@ -1,7 +1,25 @@
-import { TooltipWithBounds } from '../src';
+import React from 'react';
+import { mount } from 'enzyme';
```

Comment:
ideally we would use `shallow` over `mount`, is that possible?

Reply:
For some reason using `shallow` fails. I first suspected the fact that `TooltipWithBounds` is wrapped by a higher-order component that uses life cycle methods to be the reason but even when I use `shallow` and enable life cycle methods it still fails. 
I have seen an issue pointing out that it might be a problem with HOCs but I'm tired now. I Will read through it tomorrow and see whether I can switch mount to shallow. Its been long ever since I used Enzyme

Reply:
@williaster I found a workaround to use `shallow` over `mount`. Let me know what you think about it

Reply:
this is perfect! yeah usually you have to use `.dive()` to traverse as needed to whatever selection you are looking for. Thanks for making the change! 🙏